.github/workflows/build-prs.yml

```diff
@@ -18,6 +18,11 @@ on:
             required: false
             type: boolean
             default: false
+        headless:

```

Comment:
I tend to dislike flags you have to turn off instead of on... But I'm not sure what to call a non-headless environment in this case.